src/test/java/com/jfrog/ide/idea/integration/ApplicabilityScannerIntegrationTests.java

```diff
@@ -81,6 +81,23 @@ public void testApplicabilityScannerPythonProject() throws IOException, Interrup
         assertTrue(results.get(0).getFilePath().endsWith("main.py"));
     }
 
+    public void testApplicabilityScannerMavenProject() throws IOException, InterruptedException {
```

Comment:
I suggest changing the test name to testApplicabilityScannerJavaProject to make it clear that it's not specific to Maven, but it covers Java projects in general.